modules/ROOT/pages/6.0-release-notes-strings-and-internationalisation.adoc

```diff
@@ -0,0 +1,28 @@
+= TinyMCE 6.0 strings and internationalisaton changes
+:navtitle: TinyMCE 6.0 strings and internationalisaton changes
+:description: TinyMCE 6.0 strings and internationalisaton changes
+:keywords: releasenotes, strings, internationalisaton
+
+This section documents the single fix made to {productname} 6.0 strings and internationalisaton.
+
+* xref:strings-and-internationalisation[Strings and internationalisation]
+* xref:fixed-strings-and-internationalisation[Fixed strings and internationalisation]
+
+// tag::strings-and-internationalisation[]
+[[strings-and-internationalisation]]
+== Strings and Internationalisation
+
+
+[[fixed-strings-and-internationalisation]]
+=== Fixed strings
+
+[[color-picker-dialog]]
+==== color picker dialog
+
+The {productname} color picker presents tool tip labels (also known as aria labels) showing the name of the color the pointer is hovering over.
```

Comment:
This isn't quite correct, as `aria-labels` aren't the same thing as a tooltip. An aria-label is used to provide screenreader specific information for accessibility, whereas as a tooltip is a UI element that shows on hover.
```suggestion
The {productname} color picker presents tooltip labels showing the name of the color the pointer is hovering over.
```